docs/installation.rst

```diff
@@ -204,18 +204,33 @@ management command. Replace ``YOUR_USERNAME`` with your username and
     --password YOUR_PASSWORD
 
 With a password-enabled admin account, you can log into Django admin at
-http://localhost:8000/admin/login/
+http://localhost.org:8000/admin/login
 
 .. _enable-github-auth:
 
+Update the Sites section
+=======================================
+#. After logging in to the Django admin (an alternative is using the login ``test-super`` 
+   with password ``test-password``), scroll down to the Sites section. 
+
+#. Click on "Change". 
+
+#. Click on the entry that says ``localhost:8000``. 
+
+#. Change both the domain and display name from ``localhost:8000`` to ``localhost.org:8000``. 
+
+#. Click "Save".
+
+
+
 Enable GitHub/Google authentication (optional)
 =======================================
 Since Google's OAuth requires us to use a valid top-level-domain, we're going to use
 http://localhost.org:8000 as an example for every URL here.
 
 To automate setting Django up for social auth you can run
-``docker-compose exec web ./manage.py configure_social_auth`` and follow its steps (and
-ignore the rest of this section).
+``docker-compose exec web ./manage.py configure_github_social`` and follow its steps (and
```

Comment:
We don't have a Django management command named `configure_github_social`, so this should remain `configure_social_auth`.

Reply:
👍